benchmarks/dynamo/runner.py

```diff
@@ -264,6 +265,27 @@ def parse_args():
         default=DASHBOARD_DEFAULTS["dashboard_gh_cli_path"],
         help="Github CLI path",
     )
+    parser.add_argument(
+        "--batch_size", type=int, default=None, help="batch size for benchmarking"
+    )
+    parser.add_argument(
+        "--threads",
+        "-t",
+        type=int,
+        default=None,
+        help="number of threads to use for eager",
+    )
+    numactl_group = parser.add_argument_group("Numactl Parameters")
+    numactl_group.add_argument(
+        "--physcpubind",
+        "-C",
+        type=str,
+        default=None,
+        help="Only execute process on cpus.",
```

Comment:
"on given cpus"? Also note the default behavior in the help info?

Reply:
Updated.